### PR TITLE
include trip typescript def

### PR DESCRIPTION
**Pull request by brettguenther**

### CODE REVIEW

File: src/definitions.ts [hunk 1]
```diff
@@ -106,6 +107,18 @@ export interface RadarUser {
...
 
+export interface RadarTrip {
+  _id: string;
+  externalId: string;
+  metadata?: object;
+  destinationGeofenceTag?: string;
+  destinationGeofenceExternalId?: string;
+  mode?: string;
+  etaDistance?: number;
+  etaDuration?: number;
```
Comment (kochis) [inline src/definitions.ts:118]: Is there a reason this isn't the same as the server response?
```js
eta: {
  duration: number;
  distance: number;
}
```

Comment (brettguenther) [reply src/definitions.ts]: good catch

Comment (nickpatrick) [reply src/definitions.ts]: Yeah, here's how it's serialized right now: https://github.com/radarlabs/radar-sdk-ios/blob/master/RadarSDK/RadarTrip.m#L178 https://github.com/radarlabs/radar-sdk-android/blob/master/sdk/src/main/java/io/radar/sdk/model/RadarTrip.kt#L163

File: src/definitions.ts [hunk 2]
```diff
@@ -262,3 +274,8 @@ export interface RadarRouteDuration {
   value: number;
   text: string;
 }
+
+export interface RadarEta {
```
Comment (nickpatrick) [inline src/definitions.ts:278]: Nit: Might call this `RadarTripEta`

**Review by nickpatrick — APPROVED** [review]
Nice!

**Review by nickpatrick — APPROVED** [review]

**Pull request merged**